File: docs/source/_snippets/simulation/propagation_setup/acceleration_models/mutual_third_body_spherical_harmonic_gravity.cpp
SelectedAccelerationMap accelerationSettings;

int maximumDegreeOfIo = 12;
int maximumOrderOfIo = 12;
int maximumDegreeOfGanymede = 4;
int maximumOrderOfGanymede = 4;
int maximumDegreeOfJupiter = 4;
int maximumOrderOfJupiter = 4;

accelerationSettings[ "Io" ][ "Jupiter" ].push_back( std::make_shared< MutualSphericalHarmonicAccelerationSettings >(
    maximumDegreeOfJupiter, maximumOrderOfJupiter, maximumDegreeOfGanymede, maximumOrderOfGanymede, maximumDegreeOfIo, maximumOrderOfIo ) );

File: docs/source/_snippets/simulation/propagation_setup/acceleration_models/mutual_third_body_spherical_harmonic_gravity.py
maximum_degree_of_io = 12
maximum_order_of_io = 12
maximum_degree_of_ganymede = 4
maximum_order_of_ganymede = 4
maximum_degree_of_jupiter = 4
maximum_order_of_jupiter = 4

acceleration_settings_on_io = dict(
            Earth = [ propagation_setup.acceleration.mutual_spherical_harmonic_gravity(
            	maximum_degree_of_jupiter, maximum_order_of_jupiter, maximum_degree_of_ganymede, maximum_order_of_ganymede
            	maximum_degree_of_io, maximum_order_of_io) ] 
        )